backend/src/app.ts

```diff
@@ -53,5 +54,6 @@ app.use('/api/transactions', transactionsRouter)
 app.use('/api/users', userRouter)
 app.use('/api/qrcode', qrCodeRouter)
 app.use('/api/cron', cronRouter)
+app.use('/api/email', emailRouter)
```

Comment:
email router will stay inside the cron router. it can be pulled out, but keeping every cron job together under one router makes more sense.